functions/update-from-json-input/1.0/index.js

```diff
@@ -0,0 +1,39 @@
+import { parseAssignedProperties, fetchRecord } from '../../utils';
+
+const update = async ({
+  selectedRecord: {
+    data: { id },
+    model: { name: modelName },
+  },
+  mapping,
+  jsonInput,
+}) => {
+  const assignProperties = parseAssignedProperties(mapping, jsonInput);
+  const input = {
```

Comment:
Why do you spread this object into a new object here? Why no use assignProperties straight away